src/umv/make.rs

```diff
@@ -96,8 +98,13 @@ impl<'a, T, D> CubicSmoothingSpline<'a, T, D>
         // Solve linear system Ax = b for the 2nd derivatives
         let usol = {
             let a = {
-                let a1 = sprs_mul_s(&qtwq, s1);
-                let a2 = sprs_mul_s(&r, smooth);
+
+
+                // cannot multiply `&CsMatBase<T, usize, Vec<usize>, Vec<usize>, Vec<T>>` by `T`
+                // the trait `Mul<T>` is not implemented for `&CsMatBase<T, usize, Vec<usize>, Vec<usize>, Vec<T>>`
+
```

Comment:
The last issue blocking compilation: The Mul trait is not implemented for the CsMatBase<T> I am not sure whether it can be implemented outside the sprs crate (I tried at the bottom of `trait.rs` -- commented out.   @espdev , any ideas?